pennylane_lightning/core/_serialize.py

```diff
@@ -59,7 +59,12 @@ class QuantumScriptSerializer:
 
     # pylint: disable=import-outside-toplevel, too-many-instance-attributes, c-extension-no-member, too-many-branches, too-many-statements
     def __init__(
-        self, device_name, use_csingle: bool = False, use_mpi: bool = False, split_obs: bool = False
+        self,
+        device_name,
+        use_csingle: bool = False,
+        use_mpi: bool = False,
+        split_obs: bool = False,
+        tensor_backend: str = "mps",
```

Comment:
`"mps"` doesn't mean anything for other lightning simulators. I wonder if we have to default it to mps here. This could be defaulted to an empty string.

Reply:
I thought the same, that is why I used the variable `tensor_backend` because even for the other simulators could be confusing the variable `backend` :thinking: